README.md

```diff
@@ -1,11 +1,29 @@
 # cheri-examples
 cheri-riscv sample c programs
 
+------
 ## Build instructions with CMake
+Build al the example code with CMake using  default settings :
 1. Create a separate build directory (\<build-dir\>) outside the source directory (\<source-dir\>).
-2. Generate the build files using either **Unix Makefiles** or **Ninja**. Currently only the *RISC-V* cheri-purecap toolchain file has been created. From outside the source directory execute: 
+2. Generate the build files using either **Unix Makefiles** (*default*) or **Ninja** (specfied
+   with **-G**). Currently only the *RISC-V* cheri-purecap toolchain file has been created.
+   1. If executing from inside the build directory :
 
-   `cmake -B <build-dir> -S <source-dir> -DCMAKE_TOOLCHAIN_FILE=<source-dir>/riscv64-purecap.cmake -DSDK=<cheri-sdk-dir> -G Ninja`
-3. Build all the examples 
+   `cmake -DCMAKE_TOOLCHAIN_FILE=riscv64-purecap.cmake <relative-path-to-source-dir>`
```

Comment:
Can we make `CMAKE_TOOLCHAIN_FILE` a default too?

Reply:
I don't think we can use defaults for setting the tool chain file. If we set a default,   it will have to be done before the [`project()`](https://github.com/djichthys/cheri-examples/blob/6478241cd5c949fa75e58679c7b167c826029634/CMakeLists.txt#L3)  directive to take effect.  However, cmake then does not seem to be able to override it when we use a different file (say morello).